bundles/best_practices.rst

```diff
@@ -166,6 +166,72 @@ the ``Tests/`` directory. Tests should follow the following principles:
    A test suite must not contain ``AllTests.php`` scripts, but must rely on the
    existence of a ``phpunit.xml.dist`` file.
 
+Travis
+------
+
+A popular way to test open source bundles is by using `Travis CI`_. A good practice
+is to support at least the two latest LTS versions of Symfony. One should also test
+test latest beta release. Here is a recommended configuration file (``.travis.yml``).
+
+.. code-block:: yaml
+
+    language: php
+    sudo: false
+    cache:
+        directories:
+            - $HOME/.composer/cache/files
+            - $HOME/symfony-bridge/.phpunit
+
+    env:
+        global:
+            - TEST_COMMAND="./vendor/bin/simple-phpunit"
+            - SYMFONY_PHPUNIT_DIR="$HOME/symfony-bridge/.phpunit"
+
+    matrix:
+        fast_finish: true
+        include:
+              # Minimum supported Symfony version with the latest PHP version
+            - env: COMPOSER_FLAGS="--prefer-stable --prefer-lowest" SYMFONY_DEPRECATIONS_HELPER="weak"
```

Comment:
this syntax looks invalid to me. `env` is not a direct child of `include`

Reply:
I do not like how we pack everything to one line, but Im afraid that is how it has to be.

Reply:
It definitely works: [example](https://github.com/theofidry/AliceDataFixtures/blob/master/.travis.yml#L23-L25).

That said it might not be done on purpose and I had in some cases issues with the allow failures so you're right might be better not to promote that